doc/release-notes.md

```diff
@@ -4,3 +4,30 @@ release-notes at release time)
 Notable changes
 ===============
 
+Sprout to Sapling Migration Tool
+--------------------------------
+This release includes the addition of a tool that will enable users to migrate
+shielded funds from the Sprout pool to the Sapling pool while minimizing
+information leakage. 
+
+The migration can be enabled using the RPC `z_setmigration` or by including
+`-migration` in the `zcash.conf` file. Unless otherwise specified funds will be
+migrated to the wallet's default Sapling address; it is also possible to set the 
+receiving Sapling address using the `-migrationdestaddress` option in `zcash.conf`.
+
+See [ZIP308](https://github.com/zcash/zips/blob/master/zip-0308.rst) for full details. 
+
+
+New consensus rule: Reject blocks that violate turnstile
+--------------------------------------------------------
+In the 2.0.4 release the consensus rules were changed on testnet to enforce a
+consensus rule which marks blocks as invalid if they would lead to a turnstile
+violation in the Sprout or Shielded value pools.
+**This release enforces the consensus rule change on mainnet**
+
+The motivations and deployment details can be found in the accompanying
+[ZIP draft](https://github.com/zcash/zips/pull/210).
```

Comment:
```suggestion
[ZIP draft](https://github.com/zcash/zips/pull/210) and
[PR 3968](https://github.com/zcash/zcash/pull/3968).
```

Reply:
I made this change and removed the other reference to 3968.